UPGRADING.md

```diff
@@ -75,13 +75,20 @@ and `VerifyVoteExtensionHandler` respectively. Please see [here](https://docs.co
 for more info.
 
 
-#### Upgrade
+#### Set PreBeginBlocker
 
 **Users using `depinject` / app v2 do not need any changes, this is abstracted for them.**
+
+```diff
++ app.SetPreBeginBlocker(app.PreBeginBlocker)
```

Comment:
I think there's nothing user need to do to upgrade, it's all handled in baseapp level.